lectures/global-entrans-tuwien/index.rst

```diff
@@ -0,0 +1,62 @@
+Global Energy Transitions and Climate Policy @ TU Wien
+============================================
+
+.. figure:: ../../../../_static/tuwien_logo.png
+   :width: 240px
+   :align: right
+
+This course is part of the Master's Degree Program in `Environmental Technology & International Affairs`_ 
+held in Technical University Vienna (TU Wien) since spring 2021.
+The course is designed and delivered by `Dr. Behnam Zakeri <https://www.iiasa.ac.at/staff/zakeri>`_,
+including two lectures given by guest speakers Dr. Kavita Surana and Franziska Menten (MSc.).
+
+.. _`Environmental Technology & International Affairs` : https://www.tuwien.at/en/ace/masters-programs/msc-environmental-technology-intl-affairs?L=1
+
+Scope
+^^^^^
+
+The course offers a general overview on drivers and impacts of global energy transitions.
+Governing principles of international energy markets are reviewed briefly.
+Enablers and barriers of global energy transitions are discussed, including
+technological innovations, socio-behavioral aspects, and economic/political perspectives.
+Some contemporary topics in global energy transitions, such as, energy access and poverty,
+and energy and digitalization are discussed too. International environmental and climate agreements
+are reviewed, with a practical view on monitoring and implementation. The course is multi-disciplinary,
+suitable for students with different backgrounds, including but not limited to engineering, science, economics, and politics.
+
+
+Course material (March 2021)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|cc-by|
+
+**Lecture 1**: Global energy resources and markets
+| `pdf (2.4MB) <https://data.ene.iiasa.ac.at/zakeri/Lectures/Global-energy-transitions/Lecture-1_Global-energy-resources-and-markets.pdf>`_
+
+
+**Lecture 2**: Energy transitions: societal and individual perspectives
```

Comment:
```suggestion
**Lecture 2**: 

| Energy transitions: societal and individual perspectives
```

Reply:
Thanks @LauWien for your review. I have now added a few topics (sub-titles) under each lecture. And changed the font of the lecture title to bold. Can you please check and see if this is okay? I can put the lecture title one line below, but it's good to keep the title distinct from sub-titles, either by font or other ways.

Reply:
Thank you! I think it looks very good :)